tests/e2e/login.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect } from "@playwright/test";
+import { LoginPage } from "e2e-tests/pom/login.page";
+
+let loginPage: LoginPage;
+let validCredentials = { username: "agileway", password: "testW1se" };
+let successfullyLoggedInURL = "https://travel.agileway.net/flights/start";
+let invalidCredentials = [
+  { username: "agileway", password: "" },
+  { username: "agileway", password: "abcd1234" },
+  { username: "", password: "testW1se" },
+  { username: "abcd", password: "testW1se" },
+];
+
+test.use({ storageState: { cookies: [], origins: [] } });
+
+test.beforeEach("setup", async ({ page }) => {
+  loginPage = new LoginPage(page);
+  await loginPage.goTo();
+});
+
+test("login with valid credentials", async ({ page }) => {
+  await loginPage.fillUsername(validCredentials.username);
+  await loginPage.fillPassword(validCredentials.password);
+  await loginPage.clickSignInButton();
+
+  await expect(page).toHaveURL(successfullyLoggedInURL);
```

Comment:
after successful login would suggest to expect correct content, not URL (e.g. after success login flash-notice "Signed in!" shown